activerecord/CHANGELOG.md

```diff
@@ -1,2 +1,11 @@
+*   Add `normalizes_type` to apply normalizations to multiple attributes of the same type.
+
+    ```ruby
+    class User < ActiveRecord::Base
+      normalizes_type :string, except: :name, with: -> attribute { attribute.strip }
```

Comment:
I like the capability but I'm not sure about the API in it's current state. 

to be thought about: cc @byroot